scripts/run_tails.py

```diff
@@ -1,26 +1,23 @@
 #!/usr/bin/env python
-
-import argparse
 from astropy.coordinates import SkyCoord
 from astropy.time import Time
 from copy import deepcopy
 import datetime
-import matplotlib.pyplot as plt
```

Comment:
Were there plots before that have been removed at some point?

Reply:
Doesn't plot_stack need it?

Reply:
It does need it, but it's taken care of by the `from tails.utils import plot_stack` statement.